core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/JsonCodec.scala

```diff
@@ -29,6 +41,10 @@ object JsonCodec {
     "Google Dataproc does not support clusters with 1 non-preemptible worker. Must be 0, 2 or more.",
     List.empty
   )
```

Comment:
This is a boot disk requirement, since we take ~30GB of it for our images with the biggest set of runtime images possible. This probably should change with the change to a boot/user disk @Qi77Qi  ?